curriculum/challenges/english/10-coding-interview-prep/data-structures/create-an-es6-javascript-map.md

```diff
@@ -15,8 +15,9 @@ The new version of JavaScript provides us with a built-in Map object which provi
 - `.set(key, value)` sets a new key, value pair 
 - `.delete(key)` removes a key, value pair 
 - `.clear()` removes all key, value pairs 
-- `.entries()` returns an array of all the keys in insertion order 
-- `.values()` returns an array of all the values in insertion order
+- `.keys()` returns an iterator object that contains all the keys in insertion order 
```

Comment:
documentation says it returns an array.
Is `keys` needed for this challenge? or is it a "nice to know" addiction?

Reply:
```suggestion
- `.keys()` returns a map iterator object that contains all the keys in insertion order 
```

Reply:
Technically none of the methods are needed since the challenge can be done with the only constructor. But as far as we give an overview it would better to be the correct one.